.travis.yml

```diff
@@ -1,106 +1,62 @@
-language: php
-
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
-
-cache:
-  directories:
-    - $HOME/.composer/cache/files
+version: ~> 1.0
 
 addons:
   apt:
     packages:
       - tidy
```

Comment:
Is this still necessary? Could it be added to the shared config?

Reply:
I'm not totally sure.  I'm pretty sure it's necessary for _something_, can't remember what though

Reply:
I'm sure I've seen it required in multiple modules in the past, so perhaps just shift it to shared config and drop it here.

Reply:
ok